benchmarks/src/main/java/org/elasticsearch/benchmark/indices/resolution/IndexNameExpressionResolverBenchmark.java

```diff
@@ -63,12 +64,12 @@ public void setUp() {
         int numDataStreams = toInt(params[0]);
         int numIndices = toInt(params[1]);
 
-        Metadata.Builder mb = Metadata.builder();
+        ProjectMetadata.Builder pmb = ProjectMetadata.builder(ProjectId.DEFAULT);
```

Comment:
What is the plan with `ProjectId.DEFAULT` and `Metadata.DEFAULT_PROJECT_ID`? Are we going to keep both and use them interchangeably? I'd prefer if we move towards only having one, for consistency and clarity.